programs/uxd/tests/integration_tests/api/program_uxd/instructions/process_edit_credix_lp_depository.rs

```diff
@@ -0,0 +1,117 @@
+use anchor_lang::InstructionData;
+use anchor_lang::ToAccountMetas;
+use solana_program::instruction::Instruction;
+use solana_program::pubkey::Pubkey;
+use solana_program_test::ProgramTestContext;
+use solana_sdk::signature::Keypair;
+use solana_sdk::signer::Signer;
+
+use uxd::instructions::EditCredixLpDepositoryFields;
+use uxd::state::CredixLpDepository;
+
+use crate::integration_tests::api::program_credix;
+use crate::integration_tests::api::program_test_context;
+use crate::integration_tests::api::program_uxd;
+
+pub async fn process_edit_credix_lp_depository(
+    program_test_context: &mut ProgramTestContext,
+    payer: &Keypair,
+    authority: &Keypair,
+    collateral_mint: &Pubkey,
+    fields: &EditCredixLpDepositoryFields,
+) -> Result<(), program_test_context::ProgramTestError> {
+    // Find needed accounts
+    let controller = program_uxd::accounts::find_controller();
+    let credix_market_seeds = program_credix::accounts::find_market_seeds();
+    let credix_global_market_state =
+        program_credix::accounts::find_global_market_state(&credix_market_seeds);
+    let credix_lp_depository = program_uxd::accounts::find_credix_lp_depository(
+        collateral_mint,
+        &credix_global_market_state,
+    );
+
+    // Read state before
+    let credix_lp_depository_before =
+        program_test_context::read_account_anchor::<CredixLpDepository>(
+            program_test_context,
+            &credix_lp_depository,
+        )
+        .await?;
+
+    let redeemable_amount_under_management_cap_before =
+        credix_lp_depository_before.redeemable_amount_under_management_cap;
```

Comment:
l42-47 can be in the // THEN part around line 77, all based on l41 `redeemable_amount_under_management_cap_before` and unused till then

Reply:
sounds good ser

Reply:
done